kustomize-functions/validate-asm/Makefile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
What copyright should we use?

Reply:
Changed to the same Google copyright as the other config files.